test-suite/js/src/test/scala/org/scalajs/testsuite/javalib/util/ArraysTest.scala

```diff
@@ -25,8 +24,9 @@ object ArraysTest extends ArraysTest
  */
 trait ArraysTest extends JasmineTest {
```

Comment:
This should probably not extend `JasmineTest` anymore. Otherwise I don't understand why it uses JUnit assertions within?